### DISCUSSION TITLE
Region table does not show count of Sites

### DISCUSSION THREAD

**Discussion by hamidallaoui** [netbox-community/netbox#9049] [Q&A] [answered]
Hi All,

In Region table the count of Sites shown for each region is 0 and while click on the region name details of region are shown with correct count of Sites.

![image](https://user-images.githubusercontent.com/55854633/161932645-f7d5d433-595d-403b-9d7e-b0eddc510905.png)

![image](https://user-images.githubusercontent.com/55854633/161932801-5c174e7e-7aed-4552-bff3-f06e59025641.png)

Any idea of what can be the cause of the issue ?

Thanks & Regards,

Hamid Allaoui

**Comment by dreng** [comment]
I'm missing the "Parent" row between "Description" and "Sites". Are you using an old version of netbox?

**Comment by hamidallaoui** [comment]
I removed it from the view because I have a customized region without parent.

For info I am using netbox-3.1.8.

Thanks.

**Comment by hamidallaoui** [comment]
Hi All,

Any idea please on that issue ?


Thanks & Regards,

Hamid Allaoui

**Comment by hamidallaoui** [comment, marked as answer]
Hi All,

My issue has been fixed.

Actually I created new app with custom region (IamsRegion) and custom site (IamsSite) and in views.py file I created IamsRegionListView as:

class IamsRegionListView(generic.ObjectListView):
    queryset = IamsRegion.objects.all()
    filterset = filtersets.IamsRegionFilterSet
    filterset_form = forms.IamsRegionFilterForm
    table = tables.IamsRegionTable

while it should be:

class IamsRegionListView(generic.ObjectListView):
    queryset = IamsRegion.objects.add_related_count(
        IamsRegion.objects.all(),
        IamsSite,
        'region',
        'site_count',
        cumulative=True
    )
    filterset = filtersets.IamsRegionFilterSet
    filterset_form = forms.IamsRegionFilterForm
    table = tables.IamsRegionTable


Thanks & Regards,

Hamid Allaoui
